### PR TITLE
Bug/4508 Add change password in usermenu dropdown at homepage

### PR DESCRIPTION
**Pull request by RaghavendranC-Eaton**
Fixes # added change password in the user menu at homepage.

#### Changes proposed in this Pull Request:

-
-
-

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2023-08-04 at 11 35 51 PM

#### To Test:

- login to homepage
- select change password in the user menu dropdown
- able to see the change password dialog box

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by ektaghag-eaton — COMMENTED** [review]
After filling all fields when I clicked on Next button it is not closing dialog box

**Comment by RaghavendranC-Eaton** [comment]
> After filling in all fields when I click on the Next button it is not closing the dialogue box.
> 
> Aug-07-2023 14-40-50

@ektaghag-eaton I have fixed the change password dialog closed issue. Pls review it.

File: login-workflow/example/src/screens/ExampleHome.tsx [hunk 1]
```diff
@@ -22,20 +22,31 @@ import ExitToApp from '@mui/icons-material/ExitToApp';
...
 
 export const ExampleHome: React.FC<React.PropsWithChildren> = () => {
     const app = useApp();
     const { t, i18n } = useTranslation();
     const [open, setOpen] = useState(false);
     const navigate = useNavigate();
     const theme = useTheme();
+    const [openDialog, setOpenDialog] = React.useState(false);
+
+    const dialogClose = (): void => {
```
Comment (surajeaton) [inline login-workflow/example/src/screens/ExampleHome.tsx:43]: No need to write this function, as it is getting used only once. You can directly setOpenDialog(false)

**Review by surajeaton — COMMENTED** [review]
As discussed, we need to add hide / show change password functionality.

**Review by JeffGreiner-eaton — APPROVED** [review]
functionality looks good

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts and move the changes from new architecture to src

**Comment by RaghavendranC-Eaton** [comment]
> Please resolve the conflicts and move the changes from new architecture to src

@surajeaton Resolved conflicts.

**Comment by ektaghag-eaton** [comment]
> As discussed, we need to add hide / show change password functionality.

Seems  we are not adding any context state for `hide / show change password` 
@surajeaton , @RaghavendranC-Eaton

**Pull request merged**